subgraph.template.yaml

```diff
@@ -1311,6 +1355,44 @@ dataSources:
           handler: handleSetBigIntValueProjectConfig
       file: ./src/minter-suite-mapping.ts
   {{/minterMerkleV2Contracts}}
+{{#minterMerkleV3Contracts}}
+  - kind: ethereum/contract
+    name: 'MinterMerkleV3-{{address}}'
+    network: {{network}}
+    source:
+      address: '{{address}}'
+      abi: MinterMerkleV3
+      startBlock: {{startBlock}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - Project
+        - Contract
+        - MinterFilter
+        - Minter
+        - ProjectMinterConfiguration
+      abis:
+        - name: MinterMerkleV3
+          file: ./abis/MinterMerkleV3.json
+        - name: IFilteredMinterV0
+          file: ./abis/IFilteredMinterV0.json
+      eventHandlers:
+        - event: PricePerTokenInWeiUpdated(indexed uint256,indexed uint256)
+          handler: handlePricePerTokenInWeiUpdated
+        - event: ProjectCurrencyInfoUpdated(indexed uint256,indexed address,string)
+          handler: handleProjectCurrencyInfoUpdated
+        - event: ConfigValueSet(indexed uint256,bytes32,bytes32)
+          handler: handleSetBytesValueProjectConfig
+        - event: ConfigValueSet(indexed uint256,bytes32,bool)
+          handler: handleSetBooleanValueProjectConfig
+        - event: ConfigValueSet(indexed uint256,bytes32,uint256)
+          handler: handleSetBigIntValueProjectConfig
+        - event: DelegationRegistryUpdated(address)
+          handler: handleMerkleV3DelegationRegistryUpdated
```

Comment:
I just realized that this event actually comes from the `IFilteredMinterMerkleV1` interface.
Can you copy that ABI over to the abis directory, and import that event from the interface in `./src/minter-suite-mapping.ts` (and associated test files)?
(Basically, follow a similar pattern as `IFilteredMinterV0` and `PricePerTokenInWeiUpdated`)
This will prevent us from having to add a new handler if we ever release a new version of the Merkle minter.